### PR TITLE
Implement KittenPopo exploit fixes (and sanity changes)

### PR DESCRIPTION
**Pull request by KittenPopo**

### CODE REVIEW

File: NorthstarDedicatedTest/sigscanning.cpp [hunk 1]
```diff
@@ -5,7 +5,7 @@
 // note: sigscanning is only really intended to be used for resolving stuff like shared function definitions
 // we mostly use raw function addresses for stuff
 
-size_t GetDLLLength(HMODULE moduleHandle)
+size_t GetDLLength(HMODULE moduleHandle)
```
Comment (barnabwhy) [inline NorthstarDedicatedTest/sigscanning.cpp:8]: Should actually be DLLLength, 3 Ls cause 2 in DLL and 1 in Length

File: README.md [hunk 2]
```diff
@@ -1,12 +1,5 @@
-# NorthstarLauncher
-![Build Status](https://github.com/R2Northstar/NorthstarLauncher/actions/workflows/ci.yml/badge.svg)
+# NorthstarLauncher - ExploitFixes
```
Comment (barnabwhy) [inline README.md:1]: Don’t change readme

**Review by barnabwhy — CHANGES_REQUESTED** [review]

**Comment by barnabwhy** [comment]
other then those two things i don’t see anything wrong with this, haven’t done any testing though

File: NorthstarDedicatedTest/sigscanning.cpp (hunk 1 above)
Comment (KittenPopo) [reply NorthstarDedicatedTest/sigscanning.cpp]: Fixed, oops

**Comment by KittenPopo** [comment]
clang format be like "nice indentation, it would be a shame if i were to ruin it"

**Comment by GeckoEidechse** [comment]
> I forgot to exclude readme.md from pull request, not sure if I can change that .

Just change it back to original in another commit and we'll probably squash merge the whole thing anyway which should remove any indication the change ever happened in the first place ^^

**Comment by KittenPopo** [comment]
> Just change it back to original in another commit and we'll probably squash merge the whole thing anyway which should remove any indication the change ever happened in the first place ^^

Ok fair enough, done

**Pull request merged**